src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.tsx

```diff
@@ -246,16 +289,10 @@ const DefaultAction = ({
            */}
           <h1 className={styles.heading} data-test="actionHeading">
             <FormattedMessage
-              id={
-                (verifiedAddressesChanged &&
-                  `action.${ColonyActions.ColonyEdit}.verifiedAddresses`) ||
-                (tokensChanged &&
-                  `action.${ColonyActions.ColonyEdit}.tokens`) ||
-                roleMessageDescriptorId ||
-                'action.title'
-              }
+              id={roleMessageDescriptorId || 'action.title'}
               values={{
                 ...actionAndEventValues,
+                actionType: extendedActionType,
```

Comment:
This could be moved to line 192? It would then be consistent with the document title